src/store/mixins/createOrderedOperationMixin.ts

```diff
@@ -64,6 +64,10 @@ function queueStoreOperation(operation: Function, returnsPromise?: boolean) {
 					});
 				});
 			});
+			(<any> promise).executeNext = function() {
```

Comment:
That was the start of a bad idea that I was toying with that should not have been committed! I removed it.